includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -697,31 +707,46 @@ public function render_media_csv() {
 				array(
 					'id'          => 'delimiter',
 					'name'        => __( 'CSV Delimiter:', 'give' ),
-					'type'        => 'text',
 					'description' => __( 'In case your CSV file supports a different type of separator (or delimiter) -- like a tab or space -- you can set that here.', 'give' ),
-					'attributes'  => array( 'placeholder' => ',', 'size' => '2' ),
 					'default'     => $delimiter,
+					'type'    => 'select',
+					'options' => array(
+						'csv'  => esc_html__( 'Comma', 'give' ),
+						'tab-separated-values' => esc_html__( 'Tab', 'give' ),
+					)
 				),
 				array(
 					'id'          => 'mode',
 					'name'        => __( 'Test Mode:', 'give' ),
-					'type'        => 'checkbox',
 					'description' => __( 'Test mode allows you to preview what this import would look like without making any actual changes to your site or your database.', 'give' ),
 					'default'     => $mode,
+					'type'    => 'radio_inline',
+					'options' => array(
+						'1'  => __( 'Enabled', 'give' ),
+						'0' => __( 'Disabled', 'give' ),
+					),
 				),
 				array(
 					'id'          => 'create_user',
 					'name'        => __( 'Create WP users for new donors:', 'give' ),
-					'type'        => 'checkbox',
 					'description' => __( 'The importer can create WordPress user accounts based on the names and email addresses of the donations in your CSV file. Enable this option if you\'d like the importer to do that.', 'give' ),
 					'default'     => $create_user,
+					'type'    => 'radio_inline',
+					'options' => array(
+						'1'  => __( 'Enabled', 'give' ),
+						'0' => __( 'Disabled', 'give' ),
+					),
 				),
 				array(
 					'id'          => 'delete_csv',
 					'name'        => __( 'Delete CSV after import:', 'give' ),
-					'type'        => 'checkbox',
 					'description' => __( 'Your CSV file will be uploaded via the WordPress Media Library. It\'s a good idea to delete it after the import is finished so that your sensitive data is not accessible on the web. Disable this only if you plan to delete the file manually later.', 'give' ),
 					'default'     => $delete_csv,
+					'type'    => 'radio_inline',
+					'options' => array(
```

Comment:
`Enabled` and `Disabled` values should follow core standard.
There value should be `enabled` and `disabled`

Reply:
@ravinderk 

What I am saying is that. The whole point of the issues was related User Experiences and `enable and disable` indirectly are  `1 and 0`

I really feel okay with it? are you sure to want me to change the value to enable and disable as this value are then pass back to URL

Reply:
It's not my call, so I'll leave it up to you guys, but I believe consistency especially in the code is really important. If a 3rd party dev wanted to extend this and found that we use Enabled/Disabled everywhere, but this was the ONE setting where the value returned `0` or `1` then they'd be pretty frustrated and understandably. I wish we would have caught this before shipping the feature live so you wouldn't have to build in the backwards compatibility, but ideally we can make a note to be consistent with all Give Core settings moving forward.

Reply:
Ya @mathetos  Will keep this in mind for sure

Will change this to enable and disable that will be more good and also consistency

Reply:
@raftaar1191 Discuss this with me before doing this.

Reply:
Sure @ravinderk

Reply:
@ravinderk 
Working on this, As in the same flow that we discussed

Reply:
@ravinderk 
This is being Fixed